apps/chef_objects/src/chef_user.erl

```diff
@@ -347,8 +363,12 @@ update_from_ejson(#chef_user{} = User, UserEJson) ->
     end,
     Name = username_from_ejson(UserEJson),
     Email = ej:get({<<"email">>}, UserEJson),
-    ExternalAuthenticationUid = value_or_null({<<"external_authentication_uid">>}, UserEJson),
-    RecoveryAuthenticationEnabled = ej:get({<<"recovery_authentication_enabled">>}, UserEJson) =:= true,
```

Comment:
Steven has the use case ... I think preserving it is the right thing to do.  There is documentation on how to change it (you are supposed to use the ctl command).